libs/commons/Testutil_files.mli

```diff
@@ -0,0 +1,63 @@
+(*
+   Utilities for creating, scanning, and deleting a hierarchy
+   of test files.
+*)
+
+(*
+   We don't deal with other file types than those three.
+   File names must be normal names, i.e. not ".", "..", or "". They may not
+   contain slashes or backslashes either.
+*)
+type t =
+  | Dir of string * t list
+  | File of string * string
```

Comment:
To what correspond each string in File and Symlink? Which one is what?
Maybe File of string (* name *) * string (* content *)

and then maybe an example would be nice, like:
Dir "a" [File "a" ""; Dir "b" []; Symlink "alias" "a"].